app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
```

Comment:
The same. Looks like we do not need this class.

Reply:
Fixed